database/seeders/DatabaseSeeder.php

```
@@ -2,24 +2,176 @@
 
 class DatabaseSeeder extends Seeder
 {
-    use WithoutModelEvents;
-
-    /**
-     * Seed the application's database.
-     */
     public function run(): void
     {
-        // User::factory(10)->create();
+        User::updateOrCreate(
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\User::updateOrCreate().


(staticMethod.notFound)